ethcore/src/client/client.rs

```diff
@@ -1311,6 +1350,34 @@ impl BlockChainClient for Client {
 			earliest_state: self.state_db.lock().journal_db().earliest_era().unwrap_or(0),
 		}
 	}
+
+	fn call_contract(&self, address: Address, data: Bytes) -> Result<Bytes, String> {
+		let from = Address::default();
+		let transaction = Transaction {
+			nonce: self.latest_nonce(&from),
+			action: Action::Call(address),
+			gas: U256::from(50_000_000),
+			gas_price: U256::default(),
+			value: U256::default(),
+			data: data,
+		}.fake_sign(from);
+
+		self.call(&transaction, BlockId::Latest, Default::default())
```

Comment:
Will it not fail [here](https://github.com/ethcore/parity/blob/master/ethcore/src/executive.rs#L146) on full blocks?